api/src/main/java/org/openmrs/util/Security.java

```diff
@@ -33,14 +33,16 @@
  * OpenMRS's security class deals with the hashing of passwords.
  */
 public class Security {
-
-	private Security() {
-	}
+	
+	/**
+	 * Defined encoding to avoid using default platform charset 
+	 */
+	private static final String encoding = "UTF-8";
```

Comment:
Can you use StandardCharsets.UTF_8 instead of the above?

Reply:
I think this file is autogenerated, it keeps changing every time.